packages/wct-mocha/test/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "wct-mocha-test",
```

Comment:
add private: true so we can't accidentally publish

Reply:
Oh cool never thought of that.  Added.